### DISCUSSION TITLE
Swift Playgrounds Support

### DISCUSSION THREAD

**Discussion by powenn** [gonzalezreal/swift-markdown-ui#232] [Ideas]
It might be great to add playground support

**Comment by gonzalezreal** [comment]
Hi @powenn,

It is currently not possible, as MarkdownUI uses some C code to do the parsing (`cmark-gfm`). Sadly, Swift Playgrounds 4 only works with pure Swift language packages.

**Reply by powenn** [reply]
I see,but seems it's possible to using in inside Xcode playground,it will just shows warning and allow building it

Thanks for the response

**Reply by gonzalezreal** [reply]
As opposed to Swift Playgrounds, Xcode Playgrounds can use the complete compiler toolchain. I guess Apple limited Swift Playgrounds to make it work on iPad hardware.

**Reply by powenn** [reply]
Hi @gonzalezreal 
sorry for this mention 

Can I display local image in markdownView like using file url?

**Reply by gonzalezreal** [reply]
I haven't tried, but I am pretty sure you can use `file://` URLs, provided the path is accessible to the app. To load images from assets, you can use the built-in `AssetImageProvider` and `AssetInlineImageProvider`, as shown in the [Demo app](https://github.com/gonzalezreal/swift-markdown-ui/blob/main/Examples/Demo/Demo/ImageProvidersView.swift).

**Reply by powenn** [reply]
Okay , Thanks for your response , have a nice day
